tox.ini

```diff
@@ -1,3 +1,17 @@
 [flake8]
 max-line-length=120
 select = E,W,F,N,I
+
+[tox]
+envlist=py34,py35,flake8
+
+[tox:travis]
+3.4=py34
+3.5=py35,flake8
```

Comment:
Should we really use flake8 instead of just pyflakes? I've been thinking and I agree with Guido van Rossum when he says that style is better checked by human beings.